halconfig/settings.js

```diff
@@ -61,6 +61,11 @@ var dcos = {
 };
 var entityTagsEnabled = false;
 var netflixMode = false;
+var notifications = '{%notifications.enabled%}' === 'true'
```

Comment:
Are you using this / doesn't this flip `features.notifications`?

Reply:
Any reason to have Halyard write this as a string rather than a boolean? That's how the other flags are written.

Reply:
Ah forgot to use this.

This is to avoid compatibility issues - if you run a version of halyard that doesn't write this flag you wind up with a syntax error. All prior flags (minus pipeline templates which I believe had this problem) were in halyard before we released 1.0.

Reply:
Oh yeah. Yep.